RxSwift/Observables/ToArray.swift

```diff
@@ -10,17 +10,17 @@
 extension ObservableType {
 
     /**
-    Converts an Observable into another Observable that emits the whole sequence as a single array and then terminates.
+    Converts an Observable into a Single that emits the whole sequence as a single array and then terminates.
     
     For aggregation behavior see `reduce`.
 
     - seealso: [toArray operator on reactivex.io](http://reactivex.io/documentation/operators/to.html)
     
-    - returns: An observable sequence containing all the emitted elements as array.
+    - returns: A Single sequence containing all the emitted elements as array.
     */
     public func toArray()
-        -> Observable<[E]> {
-        return ToArray(source: self.asObservable())
+        -> Single<[E]> {
+        return PrimitiveSequence(raw: self.asObservable())
```

Comment:
That's not what I wrote :) This should be 

```suggestion
        return PrimitiveSequence(raw: ToArray(source: self.asObservable()))
```

Reply:
Oh I did write that, my mistake :O